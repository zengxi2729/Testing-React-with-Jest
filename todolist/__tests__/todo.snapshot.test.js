import Todo from '../app/todo';
import renderer from 'react-test-renderer';
import React from 'react';

test('Todo component renders the todo correctly', () => {
  const todo = { id: 1, done: false, name: 'Buy Milk' };
  const rendered = renderer.create(
    <Todo todo={todo} />
  );
  expect(rendered.toJSON()).toMatchSnapshot();
});
