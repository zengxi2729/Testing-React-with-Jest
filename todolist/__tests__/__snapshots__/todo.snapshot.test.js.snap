// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo component renders the todo correctly 1`] = `
<div
  className="todo  todo-1"
>
  <p
    className="toggle-todo"
    onClick={[Function]}
  >
    Buy Milk
  </p>
  <a
    className="delete-todo"
    href="#"
    onClick={[Function]}
  >
    Delete
  </a>
</div>
`;
